src/gmt_api.c

```diff
@@ -7145,7 +7146,8 @@ void *GMT_Read_Data (void *V_API, unsigned int family, unsigned int method, unsi
 				size_t len = strlen (file), elen;
 				char *ext = (len > 4 && strstr (file, ".cpt")) ? "" : ".cpt", *q = NULL;
 				elen = strlen (ext);
-				if ((q = gmtlib_file_unitscale (file))) q[0] = '\0';	/* Truncate modifier */
+                if ((q = gmtlib_file_unitscale (file))) q[0] = '\0';    /* Truncate modifier */
```

Comment:
Incorrect indentation due to mixed tabs and whitespaces?

Reply:
Yes, Xcode messes up my code.